__tests__/pages/telekonseling.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import TelecounselingPage, { getStaticProps } from "~/pages/telekonseling";
+import telekonseling from "~/lib/content/telekonseling";
+
+jest.mock("~/lib/content/telekonseling");
+jest.mock("next/router", () => require("next-router-mock"));
+
+describe("TelecounselingPage", () => {
+  const { supports } = telekonseling;
+
+  it("renders the title, description and breadcrumbs correctly", async () => {
+    render(<TelecounselingPage telekonseling={telekonseling} />);
+
+    const title = screen.getByRole("heading", {
+      name: /Curhat atau Telekonseling/i,
+    });
+    expect(title).toBeVisible();
+
+    const description = screen.getByText(
+      /Informasi mengenai relawan COVID-19 yang memberikan bantuan berupa layanan konsultasi psikologi, kesehatan, agama, dan lainnya yang relevan untuk membantu kesehatan mental terdampak COVID-19.$/i,
+    );
+    expect(description).toBeVisible();
+
+    const breadcrumbs = screen.getByRole("link", {
+      name: "Curhat/Telekonseling",
+    });
+    // const breadcrumbs = screen.getByText(/^Curhat atau Telekonseling$/i);
+    expect(breadcrumbs).toBeVisible();
+    expect(breadcrumbs).toHaveAttribute("href", "/telekonseling");
+    // breadcrumbs.getAttribute("href") === "telekonseling";
+  });
+
+  it("renders the topics correctly", () => {
+    render(<TelecounselingPage telekonseling={telekonseling} />);
+
+    supports.forEach((support) => {
+      expect(screen.getByText(support.support)).toBeVisible();
+
+      support.contents.forEach((content) => {
+        const link = screen.getByTestId(`next-link-${content.title}`);
+
+        expect(screen.getByText(content.title)).toBeVisible();
+        expect(link).toBeVisible();
+        expect(link).toHaveAttribute("href", content.url);
+        expect(
+          screen.getByTestId(`external-link-icon-${content.title}`),
```

Comment:
Isn't there any better selector to perform this test?
`.getByTestId()` should be our [last priority](https://testing-library.com/docs/queries/about/#priority) in querying elements.

Please use testing-playground.com Chrome extensions to help you pick better queries.

Reply:
Will do